copis/gui/panels/controller.py

```diff
@@ -21,10 +21,11 @@
 
 import wx
 import wx.lib.scrolledpanel as scrolled
+
 from pydispatch import dispatcher
-from copis.gui.wxutils import (
-    EVT_FANCY_TEXT_UPDATED_EVENT, FancyTextCtrl,
-    create_scaled_bitmap, simple_statictext)
+
+from copis.gui.wxutils import (FancyTextCtrl,
+    create_scaled_bitmap, set_dialog, simple_statictext)
```

Comment:
Makes more sense to start on a newline: have `FancyTextCtrl` on line 28

Reply:
I've also seperated the imports by types. so let's compromise like this.